pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -133,9 +133,10 @@ type MonitoringSpec struct {
 }
 
 type ClusterStatus struct {
-	State      ClusterState `json:"state,omitempty"`
-	Message    string       `json:"message,omitempty"`
-	CephStatus *CephStatus  `json:"ceph,omitempty"`
+	FinalCondition ConditionType    `json:"finalcondition,omitempty"`
+	Message        string           `json:"message,omitempty"`
+	Condition      []RookConditions `json:"conditions,omitempty"`
```

Comment:
I'd suggest this. Notice the variable is plural and type is singular.
```
Conditions  []Condition `json:"conditions,omitempty"`
```

Reply:
Okay. I'll make the changes.